client/layout/masterbar/woo-core-profiler.tsx

```diff
@@ -42,9 +44,6 @@ const WooCoreProfilerMasterbar = ( { translate }: { translate: ( text: string )
 		shouldShowNoThanks = false;
 	}
 
-	const state = useSelector( ( state ) => state );
```

Comment:
🤦‍♂️

Reply:
We have more of them, another PR coming soon.

Reply:
Following up with #103386.